pilot/docker/Dockerfile.app

```diff
@@ -1,6 +1,6 @@
 FROM scratch
-ADD client /usr/local/bin/client
```

Comment:
Nice, maybe we can also rename the binaries in the makefile. Naming binaries 'client' and server is really strange.

Reply:
it's purely for debugging and short name helps with debugging in bashless shell

Reply:
Already renamed in makefile